server.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
```

Comment:
We don't need to import `mongoose` here since we do all the Mongoose stuff inside `./data/database.js`